controllers/registeredservice_controller.go

```diff
@@ -68,27 +159,11 @@ func (r *RegisteredServiceReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	err := r.Client.Get(ctx, req.NamespacedName, &rs)
 	if err != nil && errors.IsNotFound(err) {
 		log.Info("Registered Service not found, handling delete event")
-		var sc primazaiov1alpha1.ServiceCatalog
-		err = r.Get(ctx, types.NamespacedName{
-			Name:      "primaza-service-catalog",
-			Namespace: req.NamespacedName.Namespace, // "primaza-system",
-		}, &sc)
-		if err != nil {
-			log.Error(err, "Error fetching Service Catalog")
-			return ctrl.Result{}, err
-		}
+		err = r.RemoveServiceFromCatalog(ctx, "primaza-service-catalog", req.NamespacedName.Namespace, req.Name)
```

Comment:
As it is used in multiple places, we may move this to a global unexported constant like 

```
const primazaServiceCatalog = "primaza-service-catalog"
```

Reply:
OK